custom_components/afvalwijzer/sensor.py

```diff
@@ -156,7 +166,16 @@ def update(self):
         count_today = self._config.get(CONST_COUNT_TODAY)
 
         try:
-            afvaldienst = Afvaldienst(provider, zipcode, housenumber, suffix, count_today)
+            afvaldienst = await self.hass.async_add_executor_job(
```

Comment:
```
File "/config/custom_components/afvalwijzer/sensor.py", line 169
    afvaldienst = await self.hass.async_add_executor_job(
                 ^
SyntaxError: 'await' outside async function
```

Reply:
I don't think I can fix this is a quick way. https://developers.home-assistant.io/docs/integration_fetching_data#polling-api-endpoints

Reply:
Remove this change and make the change at L129 
```self._fetch_trash_data.update()```
with
```await self._hass.async_add_executor_job(self._fetch_trash_data.update)```